bodhi/client/__init__.py

```diff
@@ -313,6 +313,11 @@ def new(user, password, url, **kwargs):
 
     for update in updates:
         try:
+            # Require severity level for a 'security' update
+            if update['type'] == 'security' and update['severity'] not in ('low', 'medium',
+                                                                           'high', 'urgent'):
+                click.echo("ERROR: must specify severity for a security update")
+                sys.exit(1)
```

Comment:
```click``` has a nice callback system that would be good to use for this. For an example, see ```_warn_if_url_and_staging_set()```.

Reply:
That might not work in case of editing an update. Because values are fetched from server for existing update. Should the user be forced to enter both ```--type``` and ```--severity```?